gps_handler.py

```diff
@@ -7,28 +7,44 @@
 # API Libraries
 import gps
 
+from process_manager import SystemState
 
 class gps_handler(object):
     def __init__(self, controller):
         self._controller = controller
         self._gpsd = gps.gps(mode=gps.WATCH_ENABLE)
+
+        self.gps_state = SystemState['ACTIVE']
+
         self._gps_poll_thread = Thread(target=self.gps_poll)
         self._gps_thread = Thread(target=self.gps_thread)
         self._gps_poll_thread.start()
         self._gps_thread.start()
 
+    def shutdown(self):
+        self.gps_state = SystemState['SHUTDOWN']
+
+    def pause(self, pause_status):
+        self.gps_state = pause_status
+
     def gps_poll(self):
-        while True:
-            next(self._gpsd)
+        while self.gps_state != SystemState['SHUTDOWN']:
+            if self.gps_state == SystemState['PAUSED']:
+                time.sleep(1)
```

Comment:
Line 4: `from time import sleep`
Can change that to `import time` for consistency with our other files. Also change the other `sleep()` calls to `time.sleep()`